lib/stytch/b2b_discovery.rb

```diff
@@ -45,6 +113,122 @@ def initialize(connection)
+      #   The type of this field is nilable `object`.
+      # sso_jit_provisioning::
+      #   The authentication setting that controls the JIT provisioning of Members when authenticating via SSO. The accepted values are:
+      #
+      #   `ALL_ALLOWED` – new Members will be automatically provisioned upon successful authentication via any of the Organization's `sso_active_connections`.
+      #
+      #   `RESTRICTED` – only new Members with SSO logins that comply with `sso_jit_provisioning_allowed_connections` can be provisioned upon authentication.
+      #
+      #   `NOT_ALLOWED` – disable JIT provisioning via SSO.
+      #
+      #   The type of this field is nilable `String`.
+      # email_allowed_domains::
+      #   An array of email domains that allow invites or JIT provisioning for new Members. This list is enforced when either `email_invites` or `email_jit_provisioning` is set to `RESTRICTED`.
+      #
+      #
+      #     Common domains such as `gmail.com` are not allowed. See the for the full list.
+      #   The type of this field is nilable list of `String`.
+      # email_jit_provisioning::
+      #   The authentication setting that controls how a new Member can be provisioned by authenticating via Email Magic Link. The accepted values are:
+      #
+      #   `RESTRICTED` – only new Members with verified emails that comply with `email_allowed_domains` can be provisioned upon authentication via Email Magic Link.
+      #
+      #   `NOT_ALLOWED` – disable JIT provisioning via Email Magic Link.
+      #
+      #   The type of this field is nilable `String`.
+      # email_invites::
+      #   The authentication setting that controls how a new Member can be invited to an organization by email. The accepted values are:
+      #
+      #   `ALL_ALLOWED` – any new Member can be invited to join via email.
+      #
+      #   `RESTRICTED` – only new Members with verified emails that comply with `email_allowed_domains` can be invited via email.
+      #
+      #   `NOT_ALLOWED` – disable email invites.
+      #
+      #   The type of this field is nilable `String`.
+      # auth_methods::
+      #   The setting that controls which authentication methods can be used by Members of an Organization. The accepted values are:
+      #
+      #   `ALL_ALLOWED` – the default setting which allows all authentication methods to be used.
+      #
+      #   `RESTRICTED` – only methods that comply with `allowed_auth_methods` can be used for authentication. This setting does not apply to Members with `is_breakglass` set to `true`.
+      #
+      #   The type of this field is nilable `String`.
+      # allowed_auth_methods::
+      #
+      #   An array of allowed authentication methods. This list is enforced when `auth_methods` is set to `RESTRICTED`.
+      #   The list's accepted values are: `sso`, `magic_link`, `password`, `google_oauth`, and `microsoft_oauth`.
+      #
+      #   The type of this field is nilable list of `String`.
```

Comment:
nit: The grammar here (and anywhere else where there's a list) sounds kind of weird but I'm not sure if there's an easy way to fix it with the codegen.